lessons/module-1/css-1.md

```diff
@@ -53,35 +53,32 @@ With your partner, answer the questions below:
 * Can you add both an id and a class to an element?
 * Which takes precedent?
 
-## CSS Combinators
-Another way to target specific relationships among elements is to use CSS combinators. We will review a few of the common selectors, but be sure to reference [the documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors#Combinators) to get a more in-depth explanation of how these rules operate.
+## CSS Combinators and Complex Selectors
+Another way to target specific relationships among elements is to use CSS combinators. We will review a few of the common selectors, but be sure to reference [the documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors#Combinators) and [this guide](https://learn.shayhowe.com/advanced-html-css/complex-selectors/) for more explanation on some of these complex selectors. 
 
-#### Descendant Combinator
-```css
-section p {
-    /* some rules */ 
-}
-```
-This combinator will target the second element that is a direct descendant of the first element. In the above example, this combinator will target all `p` tags that are descendants of a section tag.
-Syntax: space between selectors
-
-#### Adjacent Sibling Combinator
+#### Use of Commas
 ```css
-h2 + p {
+h1, h2, h3, h4 {
```

Comment:
Can you stack elements like in the css style guide? I think having as many examples of how we want things to look will be helpful to them.
```css
h1,
h2,
h3,
h4 {
/* some rules */
}